wire-ios-data-model/Source/Model/Reaction/Reaction.swift

```diff
@@ -37,11 +37,17 @@ public let ZMReactionUsersValueKey      = "users"
     @NSManaged var unicodeValue: String?
     @NSManaged var message: ZMMessage?
     @NSManaged var users: Set<ZMUser>
+    @NSManaged private var date: Date?
```

Comment:
The property name is very generic, I would give more context.

Reply:
Previously it was `firstReactionDate` would that be okay?